a3p-integration/proposals/p:upgrade-19/upgradedBoard.test.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-env node */
+// @ts-check
+
+/** @file test that the upgraded board can store and retrieve values. */
+
+import '@endo/init/legacy.js';
+import test from 'ava';
+import { evalBundles } from '@agoric/synthetic-chain';
+
+test('test upgraded board', async t => {
+  await evalBundles('submission');
```

Comment:
what bundles does this refer to? a comment might help readers like me that don't have this API swapped in

Reply:
done